src/shared/services/serializer.test.ts

```diff
@@ -99,5 +99,14 @@ describe('resultTransform', () => {
       const s = CSVSerializer(cols)
       expect(s.output()).toEqual('"{""name"":""John""}"')
     })
+    test('should escape newlines in values properly ', () => {
```

Comment:
Let's add another test case here to make sure it handles the "\r\n" variation too.